src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -148,6 +148,14 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
     register_cli_argument(scope, 'no_auto_upgrade', action='store_true', help='by doing this, extension system will not pick the highest minor version for the specified version number, and will not auto update to the latest build/revision number on any scale set updates in future.')
     register_cli_argument('{} create'.format(scope), 'generate_ssh_keys', action='store_true', help='Generate SSH public and private key files if missing', arg_group='Authentication')
 
+register_cli_argument('vmss', 'enable_autoscale', arg_group='Autoscale', action='store_true', help='Create with CPU-based autoscale settings. Assumed if one or more scaling arguments are supplied.')
```

Comment:
If you specify none of the parameters, then autoscale would not be enabled. If you want to enable it with all the defaults, you can use this flag. If you specify any of the parameters, you don't need to specify the flag (it is assumed that you are enabling it)